www/showuser

```diff
@@ -213,6 +213,14 @@ if ($rss == 'gamenews')
              $ratingHisto, $xrefs, $inrefs) =
                  getGameInfo($db, $gameid, $curuser, false,
                              $gameErrMsg, $gameErrCode);
+        $title = htmlspecialchars($title);
```

Comment:
`showuser` doesn't use most of these, so no need to escape them. Is this for "safety"?

Unused: `$license`, `$language`, `$system`, `$genre`, `$seriesname`, `$seriesnum`, `$forgiveness`.

(In the future I want getGameInfo to return a dict, so the caller can destructure just the fields it needs.)

Reply:
Yeah, I didn't want to bother to prove that `showuser` definitely _doesn't_ use any of these.

Reply:
We weren't even using the `$title`! We were only ever using the `$links`. I've updated the PR to first make `showuser` _only_ grab `$links = getGameInfo(...)[20]`, which proves that we don't need to `htmlspecialchars` anything here.